out

```diff
@@ -0,0 +1,3817 @@
+[INFO] Scanning for projects...
+[INFO] 
+[INFO] ----------------------< dev.devature:penguin-api >----------------------
+[INFO] Building penguin-api 0.0.1-SNAPSHOT
+[INFO]   from pom.xml
```

Comment:
I think you accidentally captured something here, this can be removed

Reply:
Oh yeah, I forgot about that.